rest/data/pod_test.go

```diff
@@ -17,6 +19,37 @@ func TestPodConnectorSuite(t *testing.T) {
 	suite.Run(t, s)
 }
 
+func (s *podConnectorSuite) TestCreatePod() {
```

Comment:
Can you test a bad input as well?

Reply:
You should probably add a `SetupTest` and `TeardownTest` function to clear the pod collection in the DB. Otherwise, the state will carry over between tests. The testify suites automatically recognize the setup/teardown methods and will automatically call them at the beginning/end of each test.